pyaedt/generic/configurations.py

```diff
@@ -1430,7 +1430,7 @@ def _export_materials(self, dict_out):
 
     @pyaedt_function_handler()
     def export_config(self, config_file=None, overwrite=False):
-        """Export current design properties to json file.
+        """Export current design properties to json or toml file.
         The section to be exported are defined with ``configuration.options`` class.
```

Comment:
```suggestion
        The section to be exported is defined with ``configuration.options`` class.
```
Or "The **sections** to be exported **are** defined with ``configuration.options`` class.